sql/src/main/java/io/crate/analyze/CopyToAnalyzedStatement.java

```diff
@@ -47,6 +47,10 @@
     private final boolean columnsDefined;
     @Nullable
     private final WriterProjection.CompressionType compressionType;
+    @Nullable
+    private final WriterProjection.OutputFormat outputFormat;
```

Comment:
since depending on whether columns are defined or not outputFormat has a different default value, it can be null if the format has not been defined at this stage.